tests/README.md

```diff
@@ -91,7 +91,7 @@
       -  **By:** Checking that node +h+ is available in Rancher
```

Comment:
This looks like a mistake? Could you please revert?

Reply:
I think I've done that correctly. Let me know if anything else needs to be done